modules/planning/common/frame.h

```diff
@@ -258,6 +258,10 @@ class Frame {
   std::vector<common::TrajectoryPoint>
       learning_data_adc_future_trajectory_points_;
   bool is_valid_learning_trajectory_ = false;
+
+  // hybrid model related parameters
+  double path_reference_end_pose;
+  size_t trimmed_path_bound_size;
```

Comment:
`path_reference_end_pose_` and `trimmed_path_bound_size_`

Reply:
Removed. Thank you!